pkg/kubelet/kubelet.go

```diff
@@ -1041,6 +1057,64 @@ type Kubelet struct {
 	dockerLegacyService dockershim.DockerLegacyService
 }
 
+func initializeServerCertificateManager(kubeClient clientset.Interface, kubeCfg *componentconfig.KubeletConfiguration, nodeName types.NodeName, hostname string) (certificate.Manager, error) {
```

Comment:
Changed to an array and added the hostnameoverride. Can you point me in the direction of internal and external DNS?